### PR TITLE
Changes to Environment Validator

### PR DESCRIPTION
**Pull request by owengaspard**
I like the changes @JasonLovesDoggo made, but there are some of them that I believe should be changed.

Changes I made:
1. Rename `.env.template` script and delete `.env` script.
    The script is not a file checker. The script takes the variables from the `.env.template` and puts them in a file.
2. Change the way the program checks for the `.env`
    The way `.env.template` is parsed for variables does not need to be applied to `.env`. The program simply needs to check if `.env` exists.
3. Tidying up
    I converted spaces to tabs in some files to match others, removed unnecessary comments, changed some log messages, and other things.

When I did this, I made all my changes in the `develop` branch, but I could not push it because I did not have permission lol. I had to copy everything over to the branch I should've been in, so that's why a bunch of files changed.
Files I edited:
`main.py`
`utils/checker.py`
`utils/scripts/envValidator.ps1`

### CODE REVIEW

**Comment by JasonLovesDoggo** [comment]
> removed unnecessary comments 

Those are there to help people understand the code

And don't reformat The code in a feature or

Nobody can tell what feature you added then

**Comment by owengaspard** [comment]
@JasonLovesDoggo 

I removed two comments:
`main.py`: "# Modified by JasonLovesDoggo"
`utils/checker`:  "# if .env.template exists and .env does not exist"

### Changes in `main.py`
1. Added `print_substep` - removed due to merge conflict
2. Added `checkforEnv()` function - see `utils/checker.py`

### Changes in `utils/checker.py`
1. Added `checkforEnv()` function
2. Removed the `.env` validator - this only compares `.env.template` and `.env` to see if `.env` is missing any variables. It checks for blank variables later on. Automatically appending missing variables is a bad idea because if the program messes up, it may screw up the whole `.env` file.
3. Changes to some messages - `OSError` and `FileNotFoundError` now have better output messages to better describe the issue.

### Changes made to `utils/scripts/`
1. Delete `FileGrabberenv.ps1` - only `.env.template` needs to be checked.
2. Rename `FileGrabber.ps1` to `envValidator.ps1` - This is not a file grabber in the sense that it grabs files. It parses the variables from `.env.template` to match with `.env`.

**Comment by JasonLovesDoggo** [comment]
> Delete FileGrabberenv.ps1 - only .env.template needs to be checked

No.....

**Comment by owengaspard** [comment]
It checks if `.env` exists. The PS script takes the variables from `.env.template` to match with `.env`. If any are in `.env.template` but not in `.env`, it gives the warning.

**Comment by owengaspard** [comment]
> Delete FileGrabberenv.ps1 - only .env.template needs to be checked
Let me rephrase, as I was thinking of something else when I wrote that. I did not mean to say checked. It does not check that `.env.template` is valid. As I said before, it grabs the variables in `.env.template` so that the program can use those to check `.env` in case `.env` is missing any. `.env.template` is updated, so that's why it is used to check if `.env` is valid
